fastapi-google-stt/app/services/log_script_service.py

```
@@ -45,17 +45,58 @@ async def get_scripts_by_user_id(db: AsyncSession, user_id: int) -> list | None:
+            os.makedirs(os.path.dirname(file_path), exist_ok=True)  # 디렉토리 생성
+            with open(file_path, "w", encoding="utf-8") as file:
+                file.write("")  # 빈 파일 생성
+        except Exception as e:
+            raise IOError(f"Failed to create file {file_path}: {str(e)}")
 
+    # 파일에 메시지 저장
+    try:
+        with open(file_path, "a", encoding="utf-8") as file:
+            file.write(data.message + "\n")  # 메시지를 파일에 추가
+    except Exception as e:
+        raise IOError(f"Failed to write to file {file_path}: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)